en/computer_vision/visual_inertial_odometry.md

```diff
@@ -0,0 +1,56 @@
+# Visual Intertial Odometry (VIO)
+
+*Visual Inertial Odometry* (VIO) is a [computer vision](../computer_vision/README.md) technique used for estimating the 3D *pose* (local position and orientation) and *velocity* of a moving vehicle relative to a *local* starting position.
+It is commonly used to navigate a vehicle in situations where GPS is absent or unreliable (e.g. indoors, or when flying under a bridge).
+
+VIO uses [Visual Odometry](https://en.wikipedia.org/wiki/Visual_odometry) to estimate vehicle *pose* from camera images, combined with inertial measurements from the vehicle IMU (to correct for errors associated with rapid vehicle movement resulting in poor image capture).
+
+<!-- video here -->
+
+
+
+## Supported Sensors 
+
+- [T265 Intel Realsense Tracking Camera](../peripherals/camera_t265_vio.md)
+
+
+## Setup Instructions
+
+You will need to attach your camera to the companion computer and set up ROS.
+Generally the camera should be mounted facing down. 
+If using the [T265 ](../peripherals/camera_t265_vio.md) it should also be soft-mounted.
+
+
+> **Note** ROS is not required to for VIO, but is the supported method. 
+
+These instructions explain how to set up VIO using ROS:
+- On the companion computer, install and configure [MAVROS](https://dev.px4.io/master/en/ros/mavros_installation.html) and verify it is connected successfully to the flight controller.
+- Get the Auterion [VIO bridge ROS node](https://github.com/Auterion/VIO_bridge) (this is installed as a normal ROS node).
+- Configure the [VIO bridge launch file] to reflect your camera orientation.
```

Comment:
@baumanta Which launch file? There are 3. Which part is used to configure the orientation?

Reply:
Then how do you start VIO?
Do you need to configure anything with PX4 other than that?

Reply:
the launch files say in their names what they do. The one containing sitl is for use with the PX4 sitl toolchain (it starts the vio bridge, mavros and sitl). The one bridge_mavros.launch starts the bridge and mavros which is most likely what you would need on a drone. Execpt if some other comonent is already starting mavros, then you would need the third, most basic one.

Reply:
Thanks, that was helpful